bagel/utilities/pheno_utils.py

```diff
@@ -188,17 +188,17 @@ def find_deprecated_namespaces(namespaces: list) -> list:
     return [ns for ns in namespaces if ns in DEPRECATED_NAMESPACE_PREFIXES]
 
 
-def map_categories_to_columns(data_dict: dict) -> dict[str, list]:
+def map_std_vars_to_columns(data_dict: dict) -> dict[str, list]:
     """
-    Maps all pre-defined Neurobagel categories (e.g. "Sex") to a list containing all column names (if any) that
-    have been linked to this category.
+    Maps all Neurobagel standardized variables (e.g. "Sex") to a list containing all column names (if any) that
+    have been linked to this variable.
 
-    Returns a dictionary where the keys are the aliases for Neurobagel categories and the values are lists of column names.
+    Returns a dictionary where the keys are the aliases for Neurobagel standardized variables and the values are lists of column names.
     """
     return {
-        cat_name: get_columns_about(data_dict, cat_iri)
-        for cat_name, cat_iri in mappings.NEUROBAGEL.items()
-        if get_columns_about(data_dict, cat_iri)
+        std_var_alias: get_columns_about(data_dict, std_var_iri)
```

Comment:
why is it an alias? Why not a label or a name? Alias implies synonym or alternative spellings or abstraction. I think you mean "name" here, no?